build.sbt

```diff
@@ -582,10 +596,7 @@ lazy val tests =
     )
     .settings(noPublishSettings)
     .settings(
-      // nativeOptimizerReporter := OptimizerReporter.toDirectory(
-      //   crossTarget.value),
-      // nativeLinkerReporter := LinkerReporter.toFile(
-      //   target.value / "out.dot"),
+      libraryDependencies += collectionsCompatLib,
```

Comment:
We cannot depend on `scala-collection-compat` in the unit tests, since it does not exist for Scala Native (and even if it did, we could not use it either because that would be a circular dependency).